src/modules/Login/action.js

```diff
@@ -99,6 +100,7 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
       accountInitObject.currencyCode = currencyCode
     }
     const activeWalletIds = account.activeWalletIds
+    dispatch(insertWalletIdsForProgress(activeWalletIds))
```

Comment:
can you get this information from the `accountInitAction`?

Reply:
When I tried this earlier it caused the `onAddressesChecked` callback to start firing off (eg reporting loading progress for a wallet) before we have the list of wallet ID's. For this reason I was unable to calculate the cumulative progress, which would be a major issue if any of the individual wallets immediately report 100% (only one address used in wallet) and would cause the *cumulative* progress % to never reach 100%.